### DISCUSSION TITLE
How do I get premium support when I have purchased a license from a third party authorized reseller?

### DISCUSSION THREAD

**Discussion by SzVLsY** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#968] [Q&A]
The third-party authorized reseller did not tell me the premium support email

**Comment by MatthiasKerbl** [comment]
Hi @SzVLsY,

if you bought AlDente Pro through an authorized reseller, you have to contact the support of the reseller first. If they can not solve your request, they will forward it to our support for paying customers.

Best,
Matthias

**Reply by SzVLsY** [reply]
I have talked to the reseller, but their customer service doesn't even know what I'm talking about because it is very unusual and weird to send premium support contact information ONLY in the license email. I can provide my license. I don't want to spend my time on reseller customer service as much as possible.
